internal/kafka/internal/handlers/validation_test.go

```diff
@@ -318,27 +318,110 @@ func Test_Validation_validateCloudProvider(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			gomega.RegisterTestingT(t)
+			RegisterTestingT(t)
 			validateFn := ValidateCloudProvider(&tt.arg.kafkaService, &tt.arg.kafkaRequest, tt.arg.ProviderConfig, "creating-kafka")
 			err := validateFn()
 			if !tt.want.wantErr && err != nil {
 				t.Errorf("validatedCloudProvider() expected not to throw error but threw %v", err)
 			} else if tt.want.wantErr {
-				gomega.Expect(err.Reason).To(gomega.Equal(tt.want.reason))
+				Expect(err.Reason).To(Equal(tt.want.reason))
 				return
 			}
 
-			gomega.Expect(tt.want.wantErr).To(gomega.Equal(err != nil))
+			Expect(tt.want.wantErr).To(Equal(err != nil))
 
 			if !tt.want.wantErr {
-				gomega.Expect(tt.arg.kafkaRequest.CloudProvider).To(gomega.Equal(tt.want.kafkaRequest.CloudProvider))
-				gomega.Expect(tt.arg.kafkaRequest.Region).To(gomega.Equal(tt.want.kafkaRequest.Region))
+				Expect(tt.arg.kafkaRequest.CloudProvider).To(Equal(tt.want.kafkaRequest.CloudProvider))
+				Expect(tt.arg.kafkaRequest.Region).To(Equal(tt.want.kafkaRequest.Region))
 			}
 
 		})
 	}
 }
 
+func Test_Validation_ValidateAZOption(t *testing.T) {
+	type args struct {
+		kafkaRequest dbapi.KafkaRequest
+	}
+	type result struct {
+		wantErr bool
+		reason  string
+	}
+
+	tests := []struct {
+		name string
+		args args
+		want result
+	}{
+		{
+			name: "success when standard type and mulit-AZ",
+			args: args{
+				kafkaRequest: dbapi.KafkaRequest{
+					InstanceType: "standard",
```

Comment:
same suggestion as https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/909#discussion_r846152919 to be applied throughout the added test